core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.spec.ts

```diff
@@ -121,6 +121,34 @@ describe('Contribution Opportunities backend API service', function() {
     })
   );
 
+  it('should fail to fetch the skill opportunities data ' +
```

Comment:
I think instead of writing 'it should fail' you should describe what it does if it fails. `it should do X when Y happens` pattern.

Reply:
Here and at other places

Reply:
done PTAL @aishwary023